<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>560</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TrackerTool</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Meunier Tracker Tool</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFileBrowser">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRunScript">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Run Tracker script</string>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/uqecoop2/Pictures/Icons/48px-Go-next.svg.png</normaloff>C:/Users/uqecoop2/Pictures/Icons/48px-Go-next.svg.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInput">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinDec">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>151</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>3</width>
      <height>0</height>
     </size>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Decimal Places</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Files Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFolderBrowser">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtOutputdir">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollOutput">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>521</width>
      <height>231</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>519</width>
       <height>229</height>
      </rect>
     </property>
     <widget class="QListView" name="listOutput">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>521</width>
        <height>231</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="txtOutputfile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>outfile.csv</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Filename</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Pane</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Generates quiver plots for specified plot numbers.  Plots are saved in PNG format in the specified output dir.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Create Plots</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinPlotFrom">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>3</width>
      <height>0</height>
     </size>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAllPlots">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>All Plots</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinPlotTo">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>3</width>
      <height>0</height>
     </size>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonHelp">
    <property name="geometry">
     <rect>
      <x>606</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/uqecoop2/Pictures/Icons/Help-browser.svg.png</normaloff>C:/Users/uqecoop2/Pictures/Icons/Help-browser.svg.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMatlab">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Popup each plot as a Matlab plot (limited range recommended)</string>
    </property>
    <property name="toolTipDuration">
     <number>5</number>
    </property>
    <property name="text">
     <string>Matlab plot</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxPlotly">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Popup each plot as a Matlab plot (limited range recommended)</string>
    </property>
    <property name="toolTipDuration">
     <number>5</number>
    </property>
    <property name="text">
     <string>Plotly plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Popup final plot as Matlab plot and/or Plotly plot (requires account - see Config button)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Show total plot</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
